### ISSUE TITLE
Allow for session duration filtering in cohorts

### ISSUE THREAD

**Issue by rcmarron** [PostHog/posthog#11418]
## Is your feature request related to a problem?

Users want to filter out users who visit a website and then quickly bounce.

See the [Slack thread here](https://posthogusers.slack.com/archives/C01GLBKHKQT/p1661002177236909).

## Describe the solution you'd like

Allow users to build cohorts like "users who have ever had a session > 15 seconds"

## Describe alternatives you've considered

Adding session duration filters to all insight types. (it ends up being way too complicated because it's multiple sequences of events - the session and the funnel)

## Additional context

I haven't heard this from users, but I wonder if more 'session filtering' should be added to cohorts. For example:
* Users with > 5 sessions this month
* Users who performed an event 2 times in 2 separate sessions

**Comment by EDsCODE** [comment]
Makes sense to me. I think the challenge we need to account for here is an explosion of possible filtering options on top of the ones we already have

**Comment by rcmarron** [comment]
100% agree, if we only want to support the session duration use case, this could fit in nicely with event property filters (which I think we definitely want to add). Basically, we would expose session duration as a session property in taxonomic filter (like we do in trends) when users are creating criteria like "Performed X in the last Y days". It's a bit round-about, but I think it works.

**Comment by macobo** [comment]
From the sound of this, this isn't a cohorts issue but a filtering one. As such, related issue https://github.com/PostHog/posthog/issues/7963
